Makefile

```diff
@@ -226,7 +226,8 @@ coverage-test: coverage-build
 	$(RM) out/$(BUILDTYPE)/obj.target/node_lib/gen/*.gcda
 	$(RM) out/$(BUILDTYPE)/obj.target/node_lib/src/*.gcda
 	$(RM) out/$(BUILDTYPE)/obj.target/node_lib/src/tracing/*.gcda
-	-NODE_V8_COVERAGE=out/$(BUILDTYPE)/.coverage $(MAKE) $(COVTESTS)
+	-NODE_V8_COVERAGE=out/$(BUILDTYPE)/.coverage FLAKY_TESTS="dontcare" \
```

Comment:
is this just to prevent `FLAKY_TESTS` from being overridden by an environment variable, so that `js-native-api.status` is used?

Reply:
by default it is "run" which means it will fail if the tests crashes, this means if the test is marked flaky it will still not be marked as failed.

Reply:
1) Can you double check that... Since you are not marking the tests `FLAKY`? Maybe we should keep this consistent with other CI target and just use `dontcare` as an override-able default?

2) Can we now remove the `-` prefix?

Reply:
I was on the fence on overriding the dontcare.  I think I'll remove for now. 

In terms of removing - I don't want to do that until we have a job running (similar to the linter) in the main regression job.  I want to avoid the case were  a test goes in that breaks coverage and then somebody else (people keeing coverage going) are expected to fix it. As soon as we get the coverage check as part of the regression PR then we will remove it.

Reply:
On the dontcare.  To clarify I added that when I marked them as FLAKY but that did not actually work because the builds would then always be yellow which I don't think we wanted.  Given that we'd probably we probably have to use FAIL or CRASH I removed the setting of dontcare.